### PR TITLE
Private/mmeeks/kip 1

### PR DESCRIPTION
**Pull request by mmeeks**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by mmeeks** [comment]
Latest work is on mmeeks/private/kit-in-process branch - but splitting these bits out should make that work a lot smaller and simpler after re-basing; lets get this in first.

File: kit/ForKit.cpp [hunk 1]
```diff
@@ -470,12 +470,8 @@ void forkLibreOfficeKit(const std::string& childRoot,
     // Cleanup first, to reduce disk load.
     cleanupChildren();
 
-#ifndef KIT_IN_PROCESS
-    (void) limit;
-#else
-    if (limit > 0)
+    if (!Util::isKitInProcess() && limit > 0)
```
Comment (JaumePujantell) [inline kit/ForKit.cpp:473]: The condition equivalent to the original code should be "if (Util::isKitInProcess() && limit > 0)".

Comment (mmeeks) [reply kit/ForKit.cpp:473]: ah good catch =) thanks will fix.

**Review by JaumePujantell — APPROVED** [review]
Looks good to me.

**Review by Darshan-upadhyay1110 — APPROVED** [review]
LGTM Thanks :)

**Pull request merged**